source/main.cc

```diff
@@ -450,6 +450,7 @@ int main (int argc, char *argv[])
 #ifdef ASPECT_USE_FP_EXCEPTIONS
   // enable floating point exceptions
   feenableexcept(FE_DIVBYZERO|FE_INVALID);
+  feclearexcept(FE_DIVBYZERO|FE_INVALID);
```

Comment:
Oh, I finally understand. Thanks for the patient explanation!

Maybe add a comment of the form `// clear any floating point exception bits that may be currently set` or similar. Also, wouldn't it be more logical to do that *before* you enable the exceptions? One might conceive that calling `feenableexcept` would immediately trigger the exception if the corresponding bit is currently set; that doesn't appear to be the case, but it would make sense. Clearing the bit first would be the logical recourse.